elide-core/src/main/java/com/yahoo/elide/core/RequestScope.java

```diff
@@ -94,13 +114,36 @@ public RequestScope(JsonApiDocument jsonApiDocument,
                 : Optional.of(queryParams);
 
         if (this.queryParams.isPresent()) {
+
+            /* Extract any query param that starts with 'filter' */
+            MultivaluedMap filterParams = getFilterParams(queryParams);
+
+            if (! filterParams.isEmpty()) {
+
+                /* First check to see if there is a global, cross-type filter */
+                try {
+                    globalFilterExpression = filterStrategy.parseGlobalExpression(path, filterParams);
+                } catch (ParseException e) {
+                    //TODO - perhaps aggregate this error message with any subsequent filter parsing error */
```

Comment:
We should probably honor this TODO right here. It could make debugging difficult if we only ever log the failure for a single strategy.